src/Search.js

```diff
@@ -1,23 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
+import SearchButton from "./SearchButton";
+
+const Search = ({ search }) => {
+  const [searchInput, setSearchInput] = useState("");
+  function handleSearchInput(e) {
+    setSearchInput(e.target.value);
+  }
+  function onSubmit(e) {
+    e.preventDefault();
+    if (searchInput) {
+      search(searchInput);
+    }
+  }
 
-const Search = () => {
   return (
     <div className="search">
       <div className="page-header">
         <h4 className="text-left">Search Bookings</h4>
       </div>
       <div className="row search-wrapper">
         <div className="col">
-          <form className="form-group search-box">
+          <form className="form-group search-box" onClick={onSubmit}>
             <label htmlFor="customerName">Customer name</label>
             <div className="search-row">
               <input
                 type="text"
                 id="customerName"
                 className="form-control"
                 placeholder="Customer name"
+                value={searchInput}
+                onChange={handleSearchInput}
```

Comment:
good that you're using `htmlFor` and `id` to link the label & input. Your handleSearchInput function works well